### ISSUE TITLE
Should we have a unique word for a "post"?

### ISSUE THREAD

**Issue by kizyahaha** [kizyahaha/MSB_website_design#68]
We've been using "rant" until now, but does that have too much of a negative connotation?  And does it separate our site too much to the point that people will feel unfamiliar with it?  The concept of making a "post" is universally understood.  But "creating a rant"?

**Comment by billbindi** [comment]
It feels nice based on the idea of the site, but it is very foreign, and could be very awkward. I think if we try to be too aggressive with differentiating the site, it will be alienating. Perhaps there is a better word for it, but I think post might just be the best thing right now.

**Comment by kizyahaha** [comment]
I agree that rant is not the right word, but I think we need something.  I think we want to differentiate ourselves.  We want people to feel like this is a new experience and get sucked into the game, not just another site to make posts.

**Comment by kizyahaha** [comment]
What about "box"?  Is "box" too vague?  "Build your box!"  I'm thinking about someone talking to his buddy saying, "Dude, you gotta put that in a box!" or "I'm gonna box this shit!"  It has a certain ring to it, no?  But then the whole currency thing also being boks might have to change.

**Comment by kizyahaha** [comment]
We both agree that we need something, but we don't like rant and Billy hates box with a passion.
